docs/framework/wcf/diagnostics/tracing/using-service-trace-viewer-for-viewing-correlated-traces-and-troubleshooting.md

```diff
@@ -74,7 +74,7 @@ If you examine the "Sent a message over a channel" trace, you may see the follow
        <TraceData>
           <DataItem>
              <TraceRecord xmlns="http://schemas.microsoft.com/2004/10/E2ETraceEvent/TraceRecord" Severity="Information">
-                 <TraceIdentifier>http://msdn.microsoft.com/library/System.ServiceModel.Channels.MessageSent.aspx</TraceIdentifier>
```

Comment:
Either URL is a 404, but I'm not sure if this is supposed to be HTTP like the schema URLs are

Reply:
not sure either. @Lxiamail can you help? should this perhaps link to https://docs.microsoft.com/dotnet/framework/wcf/diagnostics/tracing/system-servicemodel-channels-messagesent?

Reply:
Pinging @HongGit to see if she can help

Reply:
@mairaw I've just removed this to unblock the PR